src/types/rook.io/v1alpha1/filesystem.ts

```diff
@@ -0,0 +1,51 @@
+import {NodeAffinity, PodAffinity, PodAntiAffinity} from "../../core/v1/affinity";
+import {Toleration, ResourceRequirements} from "../../core/v1";
+import {Resource} from "../../meta";
+import {ObjectMeta} from "../../meta/v1";
+
```

Comment:
I think I'd prefer having separate packages for custom resource definitions like this that can be installed when needed. What do you think?